build/commands/lib/checkout.js

```diff
@@ -17,8 +17,6 @@ const checkout = (braveCoreRef, options) => {
       braveCoreRef = revParse.stdout.toString()
     }
 
-    let args = ['sync', '--nohooks', '--force', '--reset', '--upstream', '--revision', 'src/brave@' + braveCoreRef]
```

Comment:
I don't see where this file is used at all, can we just delete the whole thing or am I missing something?